app/presenters/tree_builder_catalog_items.rb

```diff
@@ -29,7 +29,7 @@ def x_get_tree_custom_kids(object, count_only, _options)
     # build node showing any button groups or buttons under selected CatalogItem
     @resolve ||= {}
     st = ServiceTemplate.find_by(:id => object[:id])
-    items = st.custom_button_sets + st.custom_buttons
+    items = st.custom_button_sets + st.direct_custom_buttons
```

Comment:
This sounds like an unrelated change?

(and wouldn't you also have to change `x_get_tree_aset_kids` to filter out non-direct buttons from inside the custom button set?)